src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
-
+import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Prop = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Prop> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
```

Comment:
it would be good to do a destructuring for movie
for example:
```
  const {
    title,
    description,
    imgUrl,
    imdbUrl,
  } = movie;
```

Reply:
This is a good advice!

@ibondrk don't afraid of rejecting a work